core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -120,7 +120,6 @@ var PreferencesPage = function() {
   this.setUserBio = async function(bio) {
     var inputFieldName = 'User bio input field';
     await action.clear(inputFieldName, userBioElement);
-    await saveNewChanges('User Bio');
```

Comment:
No "Saved!" Info toast will appear since the user bio will be empty when a new profile is created.
**Warning:** This is an assumption. The e2e test fail was not occurring locally.

Reply:
Hmm, should we do something like:
1. Check if the field contains something
2. Clear the field
3. If the field contained something before check that the changes were saved

Reply:
@vojtechjelinek If the field contains something when a new user is created then it is a bug.

Reply:
But `setUserBio` can be used on any user, even on user that was registered long time before and already has some bio that we want to replace (`editUserBio` only appends to the existing bio).

Reply:
If bio is already present user would press delete or backspace to clear the input box, but the clear function doesn't simulate the button press.

clear function: https://github.com/oppia/oppia/blob/72462554849e0c6960ddb4091ead086d6a93261f/core/tests/protractor_utils/PreferencesPage.js#L122

Reply:
Oh, I see, okay then it seems fine to remove it.

Reply:
Note: Pressing backspace or delete on an **empty input box** will not trigger the save function.

Reply:
@gp201 But this is a general function, right? and this function is not aware of whether it's been used by a newUser or an oldUser.

Reply:
@DubeySandeep no the function is not aware.